### PR TITLE
HW21

### PR DESCRIPTION
**Pull request by Artsapollo**

### CODE REVIEW

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -46,5 +45,9 @@
             <AppenderRef ref="PROGRAM-LOGGER-APPENDER" level="DEBUG"/>
         </Root>
 
+        <AsyncLogger name="com.mchange.v2.c3p0">
+            <level value="error"/>
+        </AsyncLogger>
+
```
Comment (Daniluko) [inline src/main/resources/log4j2.xml]: This file should not be in pr

Comment (Artsapollo) [reply src/main/resources/log4j2.xml]: Yeah, I didn't notice that when pushed

File: src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -1,50 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?> <!DOCTYPE xml>
```
Comment (SergeyPasko) [inline src/main/resources/log4j2.xml]: WHY?

File: src/main/java/Lesson16/OfficeDaoImplInterface.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package Lesson16;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Set;
+
+public class OfficeDaoImplInterface implements OfficeDao{
```
Comment (SergeyPasko) [inline src/main/java/Lesson16/OfficeDaoImplInterface.java]: Bad Name

File: src/test/java/Lesson16/OfficeDaoImplAbstractIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import org.junit.runners.MethodSorters;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+import static junit.framework.TestCase.assertTrue;
+
+@FixMethodOrder(MethodSorters.JVM)
+public class OfficeDaoImplAbstractIntegrationTest {
+    public OfficeDao officeDao = new OfficeDaoImpl();
```
Comment (SergeyPasko) [inline src/test/java/Lesson16/OfficeDaoImplAbstractIntegrationTest.java]: What is it? it isn't test of OfficeDaoImplAbstract. Also field private

File: src/test/java/Lesson16/OfficeDaoImplInterfaceIntegrationTest.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+import static junit.framework.TestCase.assertTrue;
+
+@FixMethodOrder(MethodSorters.JVM)
+public class OfficeDaoImplInterfaceIntegrationTest {
```
Comment (SergeyPasko) [inline src/test/java/Lesson16/OfficeDaoImplInterfaceIntegrationTest.java]: Same

**Pull request merged**